src/ngraph/runtime/gpu/gpu_emitter.cpp

```diff
@@ -1512,30 +1470,29 @@ namespace ngraph
                 auto softmax = static_cast<const ngraph::op::Softmax*>(node);
                 writer.block_begin();
                 {
-                    size_t softmax_index;
+                    size_t index;
                     if (softmax->get_axes().size() != args[0].get_shape().size())
                     {
                         auto& cuda_emitter =
                             external_function->get_primitive_emitter()->get_cuda_emitter();
 
-                        softmax_index = cuda_emitter->build_primitive(softmax);
+                        index = cuda_emitter->build_primitive(softmax);
                     }
                     else
                     {
                         auto& cudnn_emitter =
                             external_function->get_primitive_emitter()->get_cudnn_emitter();
 
-                        softmax_index = cudnn_emitter->build_softmax(CUDNN_SOFTMAX_FAST,
-                                                                     CUDNN_SOFTMAX_MODE_INSTANCE,
-                                                                     out[0].get_type(),
-                                                                     CUDNNEmitter::Prop::Forward,
-                                                                     args[0].get_shape());
+                        index = cudnn_emitter->build_softmax(CUDNN_SOFTMAX_FAST,
+                                                             CUDNN_SOFTMAX_MODE_INSTANCE,
+                                                             out[0].get_type(),
+                                                             CUDNNEmitter::Prop::Forward,
+                                                             args[0].get_shape());
                     }
 
-                    writer << "gpu::invoke_primitive(ctx, " << softmax_index << ", ";
-                    writer << "std::vector<void*>{" << args[0].get_name() << "}.data(), ";
-                    writer << "std::vector<void*>{" << out[0].get_name() << "}.data()";
-                    writer << ");\n";
+                    writer << "void* input[] = {" << node_names(args, {0, 0}) << "};\n";
```

Comment:
should be just `node_names(args)` here?

Reply:
Yeah, for softmax, I agree this should just be `node_names(args)`